src/database/migrations/20191225105719-create-number-of-calls.js

```diff
@@ -0,0 +1,19 @@
+'use strict';
+module.exports = {
+  up: (queryInterface, Sequelize) => {
+    return queryInterface.createTable('Number_of_calls', {
```

Comment:
`Call_number`, since this is effectively a friendly ID for the call.

Reply:
I think that Number_of_calls refers to how many times has caller called